zbuild.h

```diff
@@ -1,6 +1,34 @@
 #ifndef _ZBUILD_H
 #define _ZBUILD_H
 
+/* Determine compiler version of C Standard */
+#ifdef __STDC_VERSION__
+#  if __STDC_VERSION__ >= 199901L
+#    ifndef STDC99
+#      define STDC99
+#    endif
+#  endif
+#  if __STDC_VERSION__ >= 201112L
+#    ifndef STDC11
+#      define STDC11
+#    endif
+#  endif
+#endif
+
+/* Determine compiler support for TLS */
+#ifndef Z_TLS
+#  if __STDC_VERSION__ >= 201112 && !defined __STDC_NO_THREADS__
+#    define Z_TLS _Thread_local
+#  elif defined(__GNUC__) || defined(__SUNPRO_C)
+#    define Z_TLS __thread
+#  elif defined(_WIN32) && (defined(_MSC_VER) || defined(__ICL))
+#    define thread_local __declspec(thread)
+#  else
```

Comment:
Macro name is ```thread_local``` or ```Z_TLS```?

Reply:
Haha, thanks. I was blind to that mistake 😄